src/pages/pricing/index.tsx

```diff
@@ -12,10 +13,17 @@ import emailIsValid from 'utils/email-is-valid'
 export const getServerSideProps: GetServerSideProps = async function ({res}) {
   res.setHeader('Cache-Control', 's-maxage=1, stale-while-revalidate')
 
-  if (!process.env.NEXT_PUBLIC_STRIPE_ANNUAL_PRICE_ID)
-    throw new Error('no annual price to load 😭')
+  const {data: pricingData} = await axios.get(
+    `${process.env.NEXT_PUBLIC_AUTH_DOMAIN}/api/v1/pricing`,
+  )
+  // TODO: Is finding the annual plan by name reliable?
+  const proYearly: {price_id: string} | undefined = pricingData.plans.find(
+    (plan: any) => plan.name === 'Pro Yearly',
+  )
+
+  if (!proYearly?.price_id) throw new Error('no annual price to load 😭')
 
-  const price = await loadPrice(process.env.NEXT_PUBLIC_STRIPE_ANNUAL_PRICE_ID)
+  const price = await loadPrice(proYearly.price_id)
```

Comment:
imo the above should be moved into `loadPrice`

technically speaking we don't really need to load both from egghead api and the stripe api in this situation since the data we need is in the egghead response, **but** the data in the egghead response is all hand typed where Stripe is the legit "source of truth" so doing the double hop might make sense.

trade-off is speed to load.